OpenInActionExtension/ActionRequestHandler.swift

```diff
@@ -56,13 +60,53 @@ class ActionRequestHandler: NSObject, NSExtensionRequestHandling {
     }
 }
 
+// Search API
 private extension ActionRequestHandler {
-    func completeWithOpenUserProfile(_ username: String) {
-        doneWithResults([
-            "openURL": "mastodon://profile/\(username)"
-        ])
+    func performSearch(for url: String) {
+        guard
+            let activeAuthenticationBox = Self.appContext
+                .authenticationService
+                .mastodonAuthenticationBoxes
+                .first
+        else {
+            return doneWithResults(nil)
+        }
+        
+        Mastodon.API
+            .V2
```

Comment:
Is there a way to handle folks running old versions of Mastodon without the v2 search API?

Reply:
If someone still runs [Mastodon with a version lower than 3.0](https://github.com/mastodon/mastodon/releases/tag/v3.0.0), they have way more severe issues than a broken app :D